tests/installation/scc_registration.pm

```diff
@@ -59,6 +59,16 @@ sub run {
                 bugref       => 'bsc#1028774'
             );
         }
+        # we need to confirm 'Yes' for the keep registered scenario,
```

Comment:
Please add the poo number in the comments for later guy to understand the code. Besides, this scenario is migration via Proxy SCC with system registered while do not need to de-register on base system then register system again on migration process.